torch/csrc/jit/python/python_ivalue.h

```diff
@@ -1,9 +1,20 @@
 #pragma once
+#include <ATen/core/ivalue.h>
 #include <pybind11/pybind11.h>
 #include <torch/csrc/python_headers.h>
 
 namespace py = pybind11;
 
+// Forward-declare to avoid circular dependency with pybind_utils.h.
+
+namespace torch {
+namespace jit {
+
+at::IValue toTypeInferredIValue(py::handle input);
+
+} // namespace jit
+} // namespace torch
```

Comment:
If this is deemed too hacky, it can be addressed properly by creating a python_ivalue.cc file.